front/lib/actions/mcp_internal_actions/constants.ts

```diff
@@ -881,6 +882,40 @@ The directive should be used to display a clickable version of the agent name in
       instructions: null,
     },
   },
+  slack_bot: {
+    id: 28,
+    availability: "manual" as const,
+    allowMultipleInstances: true,
+    isRestricted: ({ featureFlags }) => {
+      return !featureFlags.includes("slack_bot_mcp");
+    },
+    isPreview: true,
+    tools_stakes: {
+      list_public_channels: "never_ask" as const,
+      list_users: "never_ask" as const,
+      get_user: "never_ask" as const,
+      post_message: "low" as const,
+      read_channel_history: "never_ask" as const,
+      read_thread_messages: "never_ask" as const,
+    },
+    timeoutMs: undefined,
+    serverInfo: {
+      name: "slack_bot",
+      version: "1.0.0",
+      description:
+        "Slack tools using workspace bot credentials. Messages and actions will appear as coming from the Dust Slack bot rather than your personal account. The Slack bot must be added to channels before it is able to post messages. Direct messages are not supported. For message search and private channel access, use the user credentials Slack tool instead.",
```

Comment:
The UI doesn't do a good job with long strings, so not sure the whole thing would be visible.